src/transformState.js

```diff
@@ -5,7 +5,22 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
-}
+  for (const action of actions) {
+    if (action.type === 'removeProperties') {
```

Comment:
It's the perfect case to use `switch` construction

also, you can use destructuring to get all properties from `action` object